cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBridgeCluster.java

```diff
@@ -167,6 +167,7 @@ public static KafkaBridgeCluster fromCrd(Reconciliation reconciliation, KafkaBri
         if (spec.getJvmOptions() != null) {
             kafkaBridgeCluster.setJavaSystemProperties(spec.getJvmOptions().getJavaSystemProperties());
         }
```

Comment:
This is not needed anymore I guess?

Reply:
The corresponding field `javaSystemProperties` is still used here (line 350):

```java
ModelUtils.javaOptions(varList, getJvmOptions(), javaSystemProperties);
```

but of course I can avoid to set the field and passing directly it from the jvm options if not null.
Taking a quick look we have this kind of way to handle it across all the other models (i.e. EntityTopicOperator as well).

Reply:
I don't know. It does already quite a lot of refactoring here. But ok I guess.